docs/source.rst

```diff
@@ -92,50 +92,47 @@ We recommend conducting all research related to your submission in Tor Browser.
 If you are unsure whether you are using Tor, you can visit the address
 https://check.torproject.org.
 
-All organizations operating SecureDrop have a *Landing Page* that provides their
-own organization-specific recommendations for using SecureDrop. We encourage you
-to consider an organization's *Landing Page* before submitting to them.
+All organizations operating SecureDrop have a landing page that provides their own organization-specific recommendations for using SecureDrop. We encourage you to consider an organization’s landing age before submitting to them.
```

Comment:
nit: minor typo here ("landing age" instead of "landing page").  I'll leave it to others whether or not we stick to the caps + italicized glossary-style "_Landing Page_" or not.

Reply:
See the related recommendation to link defined terms to the glossary: https://github.com/freedomofpress/securedrop-docs/issues/54

In this case, the glossary provides very little additional information, so I'm fine with not italicizing the term, and not linking it to the glossary.